wikidatarefisland/run.py

```diff
@@ -43,6 +44,12 @@ def main(argv, filepath):
     if 'scrape' == args.step:
         the_scraper = pipes.ScraperPipe(config, schemaorg_normalizer, schemaorg_mapper)
         simple_pump.run(the_scraper, args.input_path, args.output_path)
+        return
+
+    if 'match' == args.step:
```

Comment:
simple_match or text_match? we have other matchers designed :D

Reply:
The idea is that the other matchers (for geo, dates, etc...)  would be introduced directly into the ValueMatchers class as static methods from now on, as for the future statistical matching, I thought maybe something like `infer' or something like that, but I'm not opposed to changing the name for this. what do you think?

Reply:
nah, I think it's fine.